src/human_aware_rl/rllib/rllib.py

```diff
@@ -93,15 +94,22 @@ def action(self, state):
         my_obs = obs[self.agent_index]
 
         # Use Rllib.Policy class to compute action argmax and action probabilities
-        [action_idx], rnn_state, info = self.policy.compute_actions(
+        # The first value is action_idx, which we will recompute below so the results are stochastic
+        _, rnn_state, info = self.policy.compute_actions(
             np.array([my_obs]), self.rnn_state
         )
-        agent_action = Action.INDEX_TO_ACTION[action_idx]
 
         # Softmax in numpy to convert logits to normalized probabilities
         logits = info["action_dist_inputs"]
         action_probabilities = softmax(logits)
 
+        # The original design is stochastic across different games,
+        # Though if we are reloading from a checkpoint it would inherit the seed at that point, producing deterministic results
+        [action_idx] = random.choices(
+            [0, 1, 2, 3, 4, 5], action_probabilities[0]
```

Comment:
Minor change here to make more robust to potential future changes
```suggestion
            list(range(Action.NUM_ACTIONS)), action_probabilities[0]
```